### DISCUSSION TITLE
Plane mode / no BT connection to BG meter

### DISCUSSION THREAD

**Discussion by robertrub** [NightscoutFoundation/xDrip#3118] [Bug reports]
Not sure of this is a bug or not. I live in a white zone. When at home, I turn ON _plane mode_ and turn ON _WiFi_ and _BT_ too (it just turns OFF 4G communications).

All WiFi and BT communications work normally.

My "problem" is with the BG meter (Contour Next One) which is supposed to send BGs to xDrip via BT. If _plane mode_ is ON with _BT_ ON too, the BGs are NOT sent. If I turn OFF _plane mode_, BGs are sent. 

I might be wrong but it seems as if xDrip is looking at _plane mode_ status and not _BT status_ in order to decide to connect to the Contour or not.

![Screenshot_20231008_104955_xDrip+.jpg](https://github.com/NightscoutFoundation/xDrip/assets/76626474/0914cd97-1243-4243-98e2-2de8864c538a)

**Comment by gitanO0** [comment]
It's my understanding, unfortunately, that this is by design.
